rclcpp/include/rclcpp/node_interfaces/node_base_interface.hpp

```diff
@@ -102,6 +102,12 @@ class NodeBaseInterface
   std::shared_ptr<const rcl_node_t>
   get_shared_rcl_node_handle() const = 0;
 
+  /// Manually assert that this Node is alive (for RMW_QOS_POLICY_LIVELINESS_MANUAL_BY_NODE)
```

Comment:
nitpick: punctuation, sorry I didn't comment on each case of this before, but there were other places too.

Reply:
I believe I have fixed them all now.